source/manuals/programming-languages/css.html.md.erb

```diff
@@ -9,144 +9,71 @@ review_in: 6 months
+* [Spacing](#spacing)
+* [Vendor prefixing](#vendor-prefixing)
 * [Sass nesting](#sass-nesting)
 
 ## Whitespace
 
 Use soft tabs with a 2 space indent. This follows the conventions we use in our other projects.
 
-## Dimensions
+## Spacing
 
-Dimensions for layout should either use the [variables from the
-  frontend_toolkit][measurements.scss] or be in pixel values in multiples of `5px`.
+You should use the [GOV.UK Design System spacing scale](https://design-system.service.gov.uk/styles/spacing/).
 
-  [measurements.scss]: https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_measurements.scss
-
-  ```css
-  // Bad
-  .block {
-    margin: 28px;
-  }
-  .block {
-    padding: 12px 19px;
-  }
+### Use with deprecated libraries
 
-  // Good
-  .block {
-    margin: $gutter;
-  }
-  .block {
-    padding: 10px 20px;
-  }
-  ```
+If your project is not using the GOV.UK Design System, you should use the [variables from the
+  frontend_toolkit][measurements.scss] or use pixel values in multiples of `5px`.
 
-  Consistent measurements are better at ensuring all elements on the
-  page line up in a visually appealing manner. Pixels are used over `em` or `rem`
-  because they are consistent and predictable.
-
-### Avoid using rem sizing
-
-  Firefox allows users to change the minimum allowed font-size which
-  forces the font-size on the `html` element to be at least that minimum. Rem
-  sizes are worked out as a multiplier on the `html` elements font-size. Unless
-  you are sizing everything in rem units this can cause rems to be much
-  bigger than you are expecting. See this [pull
-  request](https://github.com/alphagov/frontend/pull/385) for details.
-
-  If you do use rems bear in mind they are not supported in IE6, 7 or 8 so you
-  will have to include a pixel value before each rem value for those browsers.
+  [measurements.scss]: https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_measurements.scss
 
-  ```css
-  font-size: 12px;
-  font-size: 1.2rem; /* This line is ignored by IE6, 7 & 8 */
-  ```
+### Include fallbacks for rem spacing
 
+Avoid using `rem` units for spacing.
 
-## Bleeding edge features
+If you are using the GOV.UK Design System without [compatibility mode](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#compatibility-mode) turned on, it will use `rem` units for font sizing with pixel fallback.
 
-When using bleeding edge features which require vendor prefixes use the [mixins
-in the frontend_toolkit][css3.scss].
+This allows users to increase the size of the text on a page by adjusting the font size in their browser settings.
 
-[css3.scss]: https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_css3.scss
+If you do need to use `rem` units, include a fallback for browsers that do not support `rem` units. This is done automatically for GOV.UK Design System font sizing.
```

Comment:
Can you give an example code snippet that uses the Design System font sizing?

Reply:
We don't really have this documented properly right now so I've just removed the duplication, hopefully if we get it documented can make this clearer in the future.